components/accept-financial-contributions/AcceptContributionsMyselfOrOrg.js

```diff
@@ -135,21 +193,95 @@ class AcceptContributionsMyself extends React.Component {
+                    />
+                  )}
+                </H2>
+              </Fragment>
+            )}
           </Flex>
+          {noOrganizationPicked && (
+            <Flex flexDirection="column" justifyContent="center" alignItems="center" my={3} minWidth={'450px'}>
+              <Flex px={3} width="100%">
+                <P my={2} fontSize="Caption" textTransform="uppercase" color="black.700">
+                  <FormattedMessage id="acceptContributions.organization.myOrgs" defaultMessage="My organizations" />
+                </P>
+                <Flex flexGrow={1} alignItems="center">
+                  <StyledHr width="100%" ml={2} />
+                </Flex>
+              </Flex>
+              {orgs.length > 0 && (
+                <Flex px={3} width="100%" flexDirection="column">
+                  {orgs.map(org => (
+                    <OrgCard
+                      alignItems="center"
+                      key={org.collective.id}
+                      my={2}
+                      onClick={() => this.setState({ organization: org.collective })}
+                    >
+                      <Avatar radius={56} collective={org.collective} />
+                      <Flex flexDirection="column" ml={3}>
+                        <P color="black.900" mb={1}>
+                          {org.collective.name}
+                        </P>
+                        <P color="black.600">@{org.collective.slug}</P>
+                      </Flex>
+                    </OrgCard>
+                  ))}
+                </Flex>
+              )}
+              <Flex px={3} width="100%">
+                <P my={2} fontSize="Caption" textTransform="uppercase" color="black.700">
+                  <FormattedMessage id="CollectivePicker.CreateNew" defaultMessage="Create new" />
+                </P>
+                <Flex flexGrow={1} alignItems="center">
+                  <StyledHr width="100%" ml={2} />
+                </Flex>
+              </Flex>
+
+              <Flex my={2} px={3} flexDirection="column" width="100%">
+                {miniForm ? (
+                  <CreateCollectiveMiniForm
+                    type="ORGANIZATION"
+                    onCancel={() => this.setState({ miniForm: false })}
+                    onSuccess={data => this.setState({ organization: data })}
+                    LoggedInUser={LoggedInUser}
+                    addLoggedInUserAsAdmin
+                    excludeAdminFields
+                  />
+                ) : (
+                  <CreateNewOrg alignItems="center" onClick={() => this.setState({ miniForm: true })}>
+                    <PlusCircle size="24" color="gray" />
+                    <P fontSize="Caption" color="black.800" ml={2}>
+                      <FormattedMessage
+                        id="acceptContributions.organization.createNewOrganization"
+                        defaultMessage="Create new organization"
```

Comment:
**[:sparkles: **Enhancement**]** We already have a string for this one. The `Organization` is title case, but according to https://docs.opencollective.com/help/about/terminology#capitalization that's how it should be.

```suggestion
                        id="Organization.CreateNew"
                        defaultMessage="Create new Organization"
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>